tests/unit/views/notifications/organization_admin_notification.feature

```diff
@@ -0,0 +1,75 @@
+      | Vetinari | owner | You invited Mustrum Ridcully to be owner of Ankh-Morpork               |
+      | Ridcully | owner | Havelock Vetinari invited you to be owner of Ankh-Morpork              |
+      | Vimes    | owner | Havelock Vetinari invited Mustrum Ridcully to be owner of Ankh-Morpork |
+      | Vetinari | admin | You invited Mustrum Ridcully to be admin of Ankh-Morpork               |
+      | Ridcully | admin | Havelock Vetinari invited you to be admin of Ankh-Morpork              |
+      | Vimes    | admin | Havelock Vetinari invited Mustrum Ridcully to be admin of Ankh-Morpork |
+
+  Scenario Outline: Ridcully accepted the invite
+    Given Vetinari invites Ridcully with role <role> to the Ankh-Morpork organization
+    When Ridcully accepts the invitation to be an admin member of the Ankh-Morpork organization
+    Then <user> gets notified with <notification_string> about the acceptance
+
+    Examples:
+      | user     | role  | notification_string                                             |
+      | Ridcully | owner | You accepted an invite to be owner of Ankh-Morpork              |
+      | Vetinari | owner | Mustrum Ridcully accepted an invite to be owner of Ankh-Morpork |
+      | Vimes    | owner | Mustrum Ridcully accepted an invite to be owner of Ankh-Morpork |
+      | Ridcully | admin | You accepted an invite to be admin of Ankh-Morpork              |
+      | Vetinari | admin | Mustrum Ridcully accepted an invite to be admin of Ankh-Morpork |
+      | Vimes    | admin | Mustrum Ridcully accepted an invite to be admin of Ankh-Morpork |
+
+  Scenario Outline: Vetinari changes Ridcully's role
+    Given Ridcully is an existing admin with roles <from_role> of the Ankh-Morpork organization
+    When Vetinari changes Ridcully's role to <to_role> in the Ankh-Morpork organization
+    Then <user> gets notified with <notification_string> about the change
+
+    Examples:
+      | user     | from_role | to_role | notification_string                                                        |
+      | Vetinari | owner     | admin   | You changed Mustrum Ridcully's role to admin of Ankh-Morpork               |
+      | Ridcully | owner     | admin   | Havelock Vetinari changed your role to admin of Ankh-Morpork               |
+      | Vimes    | owner     | admin   | Havelock Vetinari changed Mustrum Ridcully's role to admin of Ankh-Morpork |
+      | Vetinari | admin     | owner   | You changed Mustrum Ridcully's role to owner of Ankh-Morpork               |
+      | Ridcully | admin     | owner   | Havelock Vetinari changed your role to owner of Ankh-Morpork               |
+      | Vimes    | admin     | owner   | Havelock Vetinari changed Mustrum Ridcully's role to owner of Ankh-Morpork |
+
+  Scenario Outline: Vetinari removes Ridcully
+    Given Ridcully is an existing admin with roles <role> of the Ankh-Morpork organization
```

Comment:
"Given Ridcully is currently &lt;owner_or_admin&gt; of the Ankh-Morpork organization"